src/NexusMods.App.UI/Controls/Spinner/Spinner.axaml

```diff
@@ -0,0 +1,13 @@
+﻿<UserControl
+    d:DesignHeight="60"
+    d:DesignWidth="60"
+    mc:Ignorable="d"
+    x:Class="NexusMods.App.UI.Controls.Spinner.Spinner"
+    xmlns="https://github.com/avaloniaui"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+    <Arc Height="{Binding $parent.Width}"
+         Width="{Binding $parent.Height}" />
+</UserControl>
```

Comment:
You may have noticed I made a small change on ProgressRing to bind width and height to UserControl itself.
So yup, I did originally actually add it to the ring as a style, that's how that change came to be.

However I figured it would be unergonomic in the long run. You'd have to specify `Classes` every time you place either a `Spinner`, or `ProgressRing`, and these controls will be fairly common within our designs. We'd also need to change all existing `ProgressRing`(s) to use a named style.

So I figured having a reasonable default style instead would be better, so in most cases the user can just type `Spinner` or `ProgressRing` in the XAML, and that's sufficient.